app/sdil/config/RoutesWiring.scala

```diff
@@ -24,7 +24,7 @@ import play.api.routing.Router
 import sdil.actions.{AuthorisedAction, FormAction, RegisteredAction}
 import sdil.connectors._
 import sdil.controllers.test.TestingController
-import sdil.controllers.{VariationsController => UniformVariationsController, _}
+import sdil.controllers.{VariationsController => UniformVariationsController, RegistrationController => UniformRegistrationController, _}
```

Comment:
line too long, consider either using two imports or cheating and calling them `UVariationsControllers` etc.

Reply:
As the fat forms are gone for variations the first aliasing might not be necessary anyway.